src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+
+export const TextContainer = styled.div`
+  display: inline-block;
+  flex-direction: column;
+  font-weight: bold;
+`;
+
+export const TextComponent = styled.div`
+  margin-top: 10px;
+  margin-bottom: 10px;
```

Comment:
`SectionContainer`'s padding should properly take care of the spacing below this text block, so we can nix this `margin-bottom`